File: Modul 4/C/PRAK405-2310817210014-DamarjatiSuryoLaksono.c
#include <stdio.h>

int main() {
    int n, kelipatan;

    printf("Masukkan nilai n dan kelipatan : ");
    scanf("%d %d", &n, &kelipatan);

    int hasil = 0;
    int i = 1;

    while (i <= n) {
        int n_baris = 0;
        int a = i;

        while (a > 0) {
            n_baris += a * kelipatan;

            if (a == i) {
                printf("(%d * %d)", a, kelipatan);
            } else {
                printf(" + (%d * %d)", a, kelipatan);
            }

            a--;
        }

        printf(" = %d\n", n_baris);
        hasil += n_baris;
        i++;
    }

    printf("%d\n", hasil);

    return 0;
}

File: Semester-1/Pemrograman-1/Modul-4/Python/PRAK405-2310817210014-DamarjatiSuryoLaksono.py
n, kelipatan = map(int, input("").split())

hasil = 0
i = 1

while i <= n:
    n_baris = 0
    a = i

    while a > 0:
        n_baris += a * kelipatan

        if a == i:
            print(f"({a} * {kelipatan})", end="")
        else:
            print(f" + ({a} * {kelipatan})", end="")

        a -= 1

    print(f" = {n_baris}")
    hasil += n_baris
    i += 1

print(hasil)
